team-practices.md

```diff
@@ -0,0 +1,36 @@
+# FOIA Discovery Team Practices
+
+This document describes the conventions the FOIA Discovery team has agreed to work under.
+
+## Sprints
+
+We do one-week sprints, with a retro and planning meeting every Monday. We have daily standups
+with our 18F team, and daily standups with the DOJ team.
+
+## Issues
+
+We use Github + Zenhub for issue tracking. We create issues as necessary to capture stories
+and articulate requirements for specific tasks. Our kanban categories for describing the state of
+issues are:
```

Comment:
I like the idea of a short description and criteria, though I'm not sure I'm equipped to write that. Care to take a shot?